test/functional/pdo_sqlsrv/pdo_569_query_varcharmax.phpt

```diff
@@ -0,0 +1,94 @@
+--TEST--
+GitHub issue #569 - direct query on varchar max fields results in function sequence error
+--DESCRIPTION--
+Verifies that the problem is no longer reproducible.
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+require_once("MsSetup.inc");
+require_once("MsCommon_mid-refactor.inc");
+
+try {
+    $connectionInfo = "ColumnEncryption = Enabled;";
+    $conn = new PDO("sqlsrv:server = $server; database=$databaseName; $connectionInfo", $uid, $pwd);
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    $tableName = 'pdoTestTable_569';
+    dropTable($conn, $tableName);
+
+    if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+        $tsql = "CREATE TABLE $tableName ([c1] varchar(max) COLLATE Latin1_General_BIN2 ENCRYPTED WITH (ENCRYPTION_TYPE = deterministic, ALGORITHM = 'AEAD_AES_256_CBC_HMAC_SHA_256', COLUMN_ENCRYPTION_KEY = AEColumnKey))";

```

Comment:
Is there any particular reason we're restricting this to Windows?

Reply:
Because `COLUMN_ENCRYPTION_KEY = AEColumnKey` is only available in Windows
Outside we have to use Azure Key Vault